packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,298 @@
+      }
+      if (child.type === Content) {
+        acc.push(child as React.ReactElement<BentoContentProps>)
+      }
+    }
+    return acc
+  }, [])
+
+  const orderedChildren = returnOrderedChildren({styleOrder, children: validChildren, currentBreakpointSize})
+
+  return (
+    <div
+      {...colorModeProp}
+      className={clsx(
+        ...bentoItemClassArray,
+        visualAsBackground && styles[`Bento__Item--visual-as-background`],
+        className,
+      )}
+      {...rest}
+    >
+      {orderedChildren}
+    </div>
+  )
+}
+
+type BentoContentProps = {
+  leadingVisual?: ReactElement | Icon
+  padding?: Padding | ResponsivePadding
+  verticalAlign?: Align | ResponsiveAlign
+  horizontalAlign?: ReducedAlign | ResponsiveReducedAlign
+  fixedBottomLink?: boolean
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const Content = ({
+  children,
+  leadingVisual: LeadingVisual,
+  padding = 'normal',
+  verticalAlign = 'start',
+  horizontalAlign = 'start',
+  fixedBottomLink = false,
+  className,
+  ...rest
+}: BentoContentProps) => {
+  const bentoContentClassArray = [styles.Bento__Content]
+  bentoContentClassArray.push(
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'verticalAlign', verticalAlign),
+    ...returnClassBasedOnResponsiveMap('Bento__Item', 'horizontalAlign', horizontalAlign),
+    ...returnClassBasedOnResponsiveMap('Bento', 'padding', padding),
+  )
+  const HeadingChild = React.Children.toArray(children).find(
+    child => React.isValidElement(child) && child.type === _Heading,
+  )
+
+  const TextChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Text)
+  const LinkChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Link)
+  return (
+    <div className={clsx(styles[`Bento-padding--${padding}`], ...bentoContentClassArray, className)} {...rest}>
+      {React.isValidElement(LeadingVisual) &&
+        React.cloneElement(LeadingVisual as React.ReactElement<IconProps>, {
+          className: styles['Bento__Content-icon'],
+          size: LeadingVisual['size'] || 44,
+        })}
+      {React.isValidElement(HeadingChild) && React.cloneElement(HeadingChild as React.ReactElement<BentoHeadingProps>)}
+      {React.isValidElement(TextChild) &&
+        React.cloneElement(TextChild as React.ReactElement<TextProps>, {
+          variant: TextChild.props.variant || 'muted',
+          as: TextChild.props.as || 'p',
+          size: TextChild.props.size || '300',
+          className: clsx(styles['Bento_Content-text'], TextChild.props.className),
+        })}
+      {React.isValidElement(LinkChild) &&
+        React.cloneElement(LinkChild as React.ReactElement<LinkProps>, {
+          variant: LinkChild.props.variant || 'accent',
+          className: clsx(
+            styles['Bento__call-to-action'],
+            fixedBottomLink && styles['Bento__call-to-action--fixed'],
```

Comment:
Why is this prop needed? 

Ideally, the Text description should always have a `margin-bottom: 24` and the Link a `margin-top: auto` and that should work for all cases. I made a codepen to describe this: https://codepen.io/josepmartins/full/YzRBjYV

Without that margin bottom in the description, even if you set `fixedBottomLink` to `true`, you'll get cases where the text + link gets stuck

<img width="758" alt="Screenshot 2023-08-31 at 09 42 54" src="https://github.com/primer/brand/assets/912236/68cd1928-e2a7-456a-8fb7-1d77e2eaf765">

Reply:
Please let me know if I am missing some piece of information here.

Reply:
I believe that can be fixed using `align-self: end/start/center` instead, it overrides the `margin-top: auto` of the link.

I just updated the Codepen with it 👍 : https://codepen.io/josepmartins/full/YzRBjYV

Reply:
I see how you accomplished this using flex rather than a grid. This will change the logic needed to create a background image the way mine currently layers. Let me see if I can accomplish this another way before refactoring.

Reply:
I was able to solve this while keeping grid for the overlay content. https://github.com/primer/brand/pull/390/commits/c9a828d8cce5d75405cdcc1df581478f7bca0665